api/valor_api/backend/core/annotation.py

```diff
@@ -77,6 +78,12 @@ def _create_annotation(
         raster = annotation.raster.to_psql()
     if annotation.embedding:
         embedding_id = _create_embedding(db=db, value=annotation.embedding)
+    if annotation.context:
+        if isinstance(annotation.context, str):
+            context = [annotation.context]
+        else:
+            assert isinstance(annotation.context, list)
```

Comment:
- `assert` should not be used in production code as it can be ignored depending on python interpreter flags. See [link](https://stackoverflow.com/questions/944592/best-practice-for-using-assert).
- maybe add `if not all([isinstance(item, str) for item in annotation.context])` to check that the contents of `annotaiton.context` is of type `str`